lib/discourse_theme/uploader.rb

```diff
@@ -66,7 +100,8 @@ def upload_full_theme
         @theme_id
       end
     ensure
-      FileUtils.rm_f filename
+      FileUtils.rm_f(temp_dir) if temp_dir
```

Comment:
```suggestion
      FileUtils.rm_rf(temp_dir) if temp_dir
```
Does this need to be `rm_rf` since we're deleting a directory?

Reply:
Fixed as well :+1: